wordpress-mobile/gutenberg-mobile
# [FIX] Scroll position android

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1474
Gutenberg PR - https://github.com/WordPress/gutenberg/pull/18060
To test:
- In the android app, focus the first editable field you can find (example: the "What is Gutenberg?" heading block
- Scroll down, trying to reach the end of the content
- Wait and check if you still see the end of content (scroll position should not move up)

Update release notes:
